src/tests/ftest/security/cont_get_acl.py

```diff
@@ -68,7 +68,7 @@ def test_get_acl_valid(self):
                     outfile=path_to_file)
                 test_errs.extend(
                     self.error_handling(
-                        self.daos_cmd.result, "File exists"))
+                        self.daos_cmd.result, "ile exists"))
```

Comment:
typo?

Reply:
heh, no. The error string from C's errno is capitalized, but in Go it's not. This test just matches on the suffix so now it works with either implementation.